src/Tabs/index.jsx

```diff
@@ -43,19 +42,21 @@ class Tabs extends React.Component {
       const labelId = this.genLabelId(i);
 
       return (
-        <Button
-          role="tab"
-          aria-selected={selected}
-          aria-controls={this.genPanelId(i)}
-          id={labelId}
-          key={labelId}
-          onClick={() => { this.toggle(i); }}
-          className={classNames('nav-link nav-item', {
+        <li key={labelId}>
+          <button
+            role="tab"
+            aria-selected={selected}
+            aria-controls={this.genPanelId(i)}
+            id={labelId}
+            onClick={() => { this.toggle(i); }}
+            className={classNames('nav-link nav-item', {
             active: selected,
+            'border-bottom-0': !selected,
```

Comment:
Let's try to get rid of this `border-bottom-0` class name or make it optional. Generally we should avoid enforcing a style within paragon as much as we can.

Reply:
Hi Adam! Apologies for the random PR. I was trying to fix the UI of the tabs component. I'm including the border-bottom-0 className so that a bottom border is included when the tabs are not active. Otherwise it looks as though only the active tab receives a border. This is the only way I could fix the way it looks. I think @adamstankiewicz may be able to explain a little better. Let me know if you think there is a better approach

Reply:
Thanks @erinakii. It looks like since we're using the button element we are getting a background-color on the button that we don't want. Bootstrap uses anchor tags so they are fine. I would either swap out buttons for anchor tags (if that can be made okay from an accessibility perspective @wittjeff) or add a `@import "./Tabs/Tabs.scss"` to `src/index.scss` where the tabs SCSS would contain something like this:
```
.nav-tabs .nav-link {
  background: transparent;
}
```

Reply:
@abutterworth thanks for the comments. That definitely makes sense. I added the scss file to make the background transparent associated to classNames .nav-tabs and .nav-link. I figured this would be better than changing to anchor tabs. I believe for accessibility it makes more sense to leave them as buttons, since we are not navigating away from the page. 

Let me know what you think! 
@adamstankiewicz @wittjeff